backend/source/archive/models/gpx.py

```diff
@@ -32,7 +32,7 @@ def entry_from_point(self, point) -> Entry:
 
     def extract_entries(self) -> Generator[Entry, None, None]:
         for gpx_file in self.get_archive_files():
-            gpx = gpxpy.parse(gpx_file)
+            gpx = gpxpy.parse(open(gpx_file, 'r'))
```

Comment:
Can you use gpx_file.open() for consistency? get_archive_files returns Path objects

Reply:
The 'r' is implicit, but open()ing the file is what was missing.